app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx

```diff
@@ -219,6 +225,24 @@ function EditableEntry(props: Props) {
         verifiedBy?.map((v) => +v.id) ?? []
     ), [verifiedBy]);
 
+    const handleAddButtonClick = useCallback((entryIdToAdd: string, sectionId?: string) => {
+        const link = generatePath(routes.entryEdit.path, {
+            projectId,
+            leadId,
+        });
+        if (sectionId) {
+            history.push(`${link}#/primary-tagging`, {
+                entryId: entryIdToAdd,
+                sectionId,
+            });
+        } else {
+            history.push(`${link}#/secondary-tagging`, {
+                entryId: entryIdToAdd,
+                sectionId,
+            });
+        }
```

Comment:
We can easily switch between primary and secondary tagging using ternay operator on sectionId

Reply:
ok